test/audience.robot

```diff
@@ -12,11 +12,14 @@ Force Tags   audience
 *** Test cases ***
 
 Token with random audience is rejected
+    ${token}   Get token
     ${uuid}   Generate UUID
-    ${token}   Get token   scope=-s openid   opts=--aud=${uuid}
     ${url}   SE URL   audience-test-${uuid}
+    ${rc}   ${out}   Curl Put Success   /etc/services  ${url}
+    Should Match Regexp   ${out}   20[01].*
+    ${token}   Get token   scope=-s openid   opts=--aud=${uuid}
     ${rc}   ${out}   Curl Error   ${url}
-    Should Contain   ${out}   401
+    ${ret}   Should Match Regexp   ${out}   40[0-9].*
```

Comment:
```suggestion
    ${ret}   Should Match Regexp   ${out}   40[134]
```
For the motivation, see https://github.com/indigo-iam/wlcg-jwt-compliance-tests/issues/12#issuecomment-1000193411.
I don't think that `.*` is needed at the end of the regex, but it needs testing.